### ISSUE TITLE
How to visualize the ThunderGBM?

### ISSUE THREAD

**Issue by DarkoAlexander** [Xtra-Computing/thundergbm#77]
Is there a way to see what is under the hood of the random forest? Is there a way to get the parameters of a trained model? Or maybe you can get the individual decisions trees as a nested-if-statement?  

Thanks alot!

**Comment by Kurt-Liuhf** [comment]
Hi @DarkoAlexander, currently feature of visualzation is not supported in ThunderGBM, and we are considering to add this feature in the future. Also, we will be glad if you can make a contribution.

**Comment by DarkoAlexander** [comment]
You say there is no feature of visualization. Is there also no way to get the parameters of the trained model? Is there no way to get the numerical values for each node in each tree in the forest?

**Comment by Kurt-Liuhf** [comment]
Right, there is no way to get the parameters of the trained model (or numerical values of each node) in Python for now. Maybe you can refer to [this](https://github.com/Xtra-Computing/thundergbm/blob/35ddc80c6fc2d072478f8505312fd6fc34dcdd9c/python/thundergbm/thundergbm.py#L280) and implement your own method.
